src/pages/en/core-concepts/framework-components.md

```diff
@@ -119,6 +119,26 @@ import MyVueComponent from '../components/MyVueComponent.vue';
 Only **Astro** components (`.astro`) can contain components from multiple frameworks.
 :::
 
+## Passing Props to Framework Components
+
+You can pass props from Astro components to framework components:
+
+```astro title="src/pages/frameworks-props.astro"
+---
+import TodoList from '../components/TodoList.jsx';
+import Counter from '../components/Counter.svelte';
+---
+<div>
+  <TodoList initialTodos={["learn Astro", "review PRs"]} />
+  <Counter startingCount={1} />
+</div>
+```
+
+:::caution
+A function can be passed as a prop to a framework component, but the function can only be called on the server. This is because functions can't be _serialized_ (transferred from the server to the client) by Astro. If the component is hydrated, an error will occur when it tries to call the function.
```

Comment:
Is this a case of "You probably don't want to do this, because it doesn't work the way you might think it does/the way it does in SPAs?" I hesitate to start the note with "You CAN pass it... " when the result is very likely to be "... but it's not gonna work."

If you can't even describe a good use case, what if we phrase this to advise against doing this? I still don't get a good sense from reading this when it actually would be a working thing for me to do.

So, what is the caution trying to caution us ABOUT - it doesn't work the way you think it will? It's so rarely going to be the thing you want that you probably should try a different pattern in your code? I think that could still be clearer here.